packages/ui/src/validators/hooks/useStakingStatistics.tsx

```diff
@@ -1,14 +1,14 @@
 import { BN } from '@polkadot/util'
-import { useMemo } from 'react'
+import { useMemo, useState } from 'react'
 import { combineLatest, map } from 'rxjs'
 
 import { useApi } from '@/api/hooks/useApi'
-import { ERA_DURATION } from '@/common/constants'
+import { useFirstObservableValue } from '@/common/hooks/useFirstObservableValue'
 import { useObservable } from '@/common/hooks/useObservable'
 
 export const useStakingStatistics = () => {
   const { api } = useApi()
-  const activeEra = useObservable(
+  const activeEra = useFirstObservableValue(
```

Comment:
Actually I think this should be a `useObservable`. It should update only every 6h. I'm thinking of even passing it to `useValidatorsWithDetails` (in a followup PR) to update the queries there too.